cherrypy/_cpreqbody.py

```diff
@@ -802,7 +843,13 @@ def read(self, size=None, fp_out=None):
             # Check lengths.
             self.bytes_read += datalen
             if self.maxbytes and self.bytes_read > self.maxbytes:
-                raise cherrypy.HTTPError(413)
+                if self.maxbytes_extend:
+                    print("extending maxbytes")
```

Comment:
Uh oh. That was a debug print statement that I was using(So I could do a search for print statements and delete it easier than searching through the log calls).

Looks like I forgot the last step.